.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2024 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
+name: Update Versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The new version to use for the extension (ex: 1.2.3).'
+        type: string
+        required: true
+
+      branch:
+        description: 'The target branch where the version update will be applied and the pull request will be merged into.'
+        type: string
+        required: true
+
+      core-dependency:
+        description: 'If a version is provided, update AEPCore dependency in podspec and Package.swift.'
+        type: string
+        required: false
+        default: ''
+
+      edge-dependency:
+        description: 'If a version is provided, update AEPEdge dependency in podspec and Package.swift.'
+        type: string
+        required: false
+        default: ''
+
+jobs:
+  update-versions:
+    permissions:
+      contents: write
+      pull-requests: write
+    uses: adobe/aepsdk-commons/.github/workflows/versions.yml@gha-ios-5.0.0
+    with:
+      version: ${{ github.event.inputs.version }}
+      branch: ${{ github.event.inputs.branch }}
+      dependencies: AEPCore ${{ github.event.inputs.core-dependency }}, AEPEdge ${{ github.event.inputs.edge-dependency }}
+      paths: Package.swift:swift_spm, AEPEdge.podspec, AEPEdge.xcodeproj/project.pbxproj, Sources/ConsentConstants.swift
```

Comment:
You missed updating to AEPEdgeConsent
AEPEdgeConsent.podspec
AEPEdgeConsent.xcodeproj/project.pbxproj